prerequisites.rst

```diff
@@ -1,35 +1,280 @@
+this is because it relies on the operating system Debian GNU/Linux [9]_
 
-* **CPU**: Any x86-64 or ARM compatible processor
-* **RAM**: 1 GiB capacity
-* **HDD**:
+Any system has software and hardware requirements, also minimal
+ones and recommended ones.
 
-  * **System Drive**: min. 4 GiB capacity (plus the capacity of the RAM)
-  * **Data Drive**: capacity according to your needs
+Hardware requirements
+---------------------
 
-.. note::
-   The whole disc will be occupied by the system and swap space [1]_, so size
-   doesn't matter so much. Data storage on the system disc is not supported.
+======  ================  =========  ==========  ==================================
+ Item    supported         Minimal     Best       Recommendation
+======  ================  =========  ==========  ==================================
+ DRIV    SSD/HDD/USB...     1,any     2,KIOXIA    2 disks: Seagate Firecuda, WD Black, IronWolf
+ RAM     1GiB+ any          1GiB       4Gig       8GiB+ dual channel DDR4/DDR3
+ NIC     WiFi/Ether/USB     any       10Mb NIC    1GiB NIC or 10Gb NICs: SFP fiber
+ CPU     arm,x86,x64        32bit      64bit      Intel Dual Core, AMD Ryzen
+======  ================  =========  ==========  ==================================
 
-.. note::
-   Take into account in your considerations about the size of the system disk
-   that various plugins such as `openmediavault-podman` or `openmediavault-k8s`
-   store their data (e.g. container images) on the system disk.
+The storage drives (DRIV)
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Spinning Harddisk, SSD [2]_, Disk-on-Module [3]_, CompactFlash [4]_ or USB thumb
-drive [5]_ type drives can be used as system drive.
+This is the key of the system. |omv| supports any drive hardware and any drive 
+interface (SATA, ISE, SCSI, USB, SERIAL) but information and management will
+depend on the disk drives supported by the Debian operating system.
+**Supported Debian OS versions can be checked in the** :doc:`releases section </releases>`.
+
+However, the manufacturers to consider depend on the type of disks. 
+For HDDs, Seagate FireCuda disks, Western Digital Black disks, 
+and for SSDs, KIOXIA or Toshiba are the best for. In case of USB or EMMC, 
+companies are reluctant to provide information, so avoid the Samsung brand.
+
+Modern storage drives have firmware inside that reports several attributes.
+Any storage drives is supported but |omv| only supports SMART for those
+disks that are connected to an HBA in pass-through mode.
+
+The system manages two types of storage unit classification:
+
+System drive storage (SDS)
+  The Storage Disk Drive(s) used to put the system program files (by partitions), 
+  **this system drive (that is in fact a couple or more partitions) 
+  cannot officially be used for shared resources or as user data drive**.
+  What supported brands and sizes are covered in the software requirements section below.
```

Comment:
Remove this because it has also been removed above.
```suggestion
```

Reply:
removed with commit 4ed84dc4d823e4f081b3db238bd701fdfe2d18a7